src/renderers/webgpu/WebGPUBackend.js

```diff
@@ -256,7 +255,7 @@ class WebGPUBackend extends Backend {
 
 			const alphaMode = parameters.alpha ? 'premultiplied' : 'opaque';
 
-			const toneMappingMode = ColorManagement.getToneMappingMode( this.renderer.outputColorSpace );
+			const toneMappingMode = this.parameters.outputType === HalfFloatType ? 'extended' : 'standard';
```

Comment:
@sunag  `this.` not needed.

Reply:
Done 5ebb122405c9cd97b2947f0df149b50a87b68fa9 👍